.github/workflows/docs.yml

```diff
@@ -79,8 +79,13 @@ jobs:
           hatch run docs:mike deploy --push --update-aliases --title "dev (${SHORT_SHA})" dev
 
       - name: Build & deploy docs for a new tag
-        if: github.ref_type == 'tag' && github.event_name == 'push'
+        if: github.event_name == 'push' && github.ref_type == 'tag'
```

Comment:
Switched the order so all IFs start with `github.event_name` check

Reply:
If you look, gh-pages branch was updated, but see #650. This step was also run so I think it is correct. I expect the failure on the last tag https://github.com/EmilStenstrom/django-components/actions/runs/10708912382 was due to the one-time clean up and will not happen again. The run shows that the triggers were correct and don't need to be changed.

Reply:
The debug section in the linked run also shows that both vars had the expected value.